cmd/commands/cmd_import_mission_control.go

```diff
@@ -98,3 +101,161 @@ func importMissionControl(ctx *cli.Context) error {
 	_, err = client.XImportMissionControl(rpcCtx, req)
 	return err
 }
+
+var loadMissionControlCommand = cli.Command{
+	Name:     "loadmc",
+	Category: "Payments",
+	Usage: "Load mission control results to the internal mission " +
+		"control state from a file produced by querymc with the " +
+		"option to shift timestamps. Note that this data is not " +
+		"persisted across restarts.",
+	Action: actionDecorator(loadMissionControl),
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "mcdatapath",
+			Usage: "The path to the querymc output file (json).",
+		},
+		cli.BoolFlag{
+			Name:  "discard",
+			Usage: "Discards current mission control data.",
+		},
+		cli.StringFlag{
+			Name: "timeoffset",
+			Usage: "Time offset to add to all timestamps. " +
```

Comment:
Should we explain what this is useful for?

Reply:
Right, added an explanation.